### DISCUSSION TITLE
How to stop being pestered to install extensions?

### DISCUSSION THREAD

**Discussion by ralfmicroseptec** [VSCodium/vscodium#1830] [General]
There are certain files I regularly maintain in YAML format.  These files are not particularly complicated, so I am fine managing them manually without a lot of plugin support.  The main problem I have is that VSCodium keeps popping up this dialog:
<img width="259" alt="Screenshot 2024-03-14 at 20 32 04" src="https://github.com/VSCodium/vscodium/assets/83094196/153c4175-fa7a-4fed-8252-7d7e93a24775">

I have no interest in installing a YAML extension, but I have yet to figure out how to get VSCodium to stop pestering me to install one.  Is there a way to get VSCodium to stop trying to shove unwanted plugins down my throat?

**Comment by daiyam** [comment]
`"extensions.ignoreRecommendations": true,` (https://code.visualstudio.com/docs/editor/extension-marketplace#_can-i-stop-vs-code-from-providing-extension-recommendations)
